### DISCUSSION TITLE
High Severity warnings with @types d3-color

### DISCUSSION THREAD

**Discussion by amcd-dev** [tremorlabs/tremor-npm#133] [Q&A]
Apologies if this is a dumb point to raise, but when installing Tremor with my newly created next.js app npm is giving 5 high severity vulnerabilities related to this -  https://github.com/advisories/GHSA-36jr-mh4h-2g58

When I force update it breaks and removes the tremor.css (and possibly more?) file.

Any idea how to work around this?

**Comment by mitrotasios** [comment]
hi @amcd-dev, this is an https://github.com/recharts/recharts/issues/3012 in Recharts (the chart library we are using under the hood). I hope it gets resolved soon! For more information please refer to this Issue #159

**Reply by mitrotasios** [reply]
should be fixed with `v1.6.0`
